.storybook/preview.js

```diff
@@ -8,4 +15,14 @@ export const parameters = {
       order: ["Components", "Design", "*"],
     },
   },
-}
+  docs: {
+    components: {
+      wrapper: props => <Content>{props.children}</Content>,
+      h1: props => <Heading {...props} level={1} />,
```

Comment:
Does this break the clickable/copyable heading URLs? @darryltec mentioned that might be the reason those don't work in our mobile Storybook instance.

Reply:
most likely, i'll bring em back on my PR. Seems like we have to do some custom bits on it to make it play nice

Reply:
We could potentially also just drop the CSS into the `preview-head` 🤷 

I think the crucial adjustments would be

`{ color: var(--color-heading); font-family: var(--typography--fontFamily-display); }`

Reply:
They use semantic tags anyway so we probably could just target those. The alternative is yeah, implement our own link anchor